docs/torch/node_class.rst

```diff
@@ -2,11 +2,15 @@
 Node Classification with GAT
 ****************************
 
-In this guide we use a pre-built `Graph Attention Network(GAT) <https://arxiv.org/abs/1710.10903>`_ model to classify nodes in the `Cora dataset <https://graphsandnetworks.com/the-cora-dataset/>`_. Readers can expect an understanding of the DeepGNN experiment flow and details on model design.
+In this guide we use a pre-built `Graph Attention Network(GAT) <https://arxiv.org/abs/1710.10903>`_ model
+to classify nodes in the `Cora dataset <https://graphsandnetworks.com/the-cora-dataset/>`_. Readers can
+expect an understanding of the DeepGNN experiment flow and details on model design.
 
 Cora Dataset
 ============
-The Cora dataset consists of 2708 scientific publications represented as nodes interconnected by 5429 reference links represented as edges. Each paper is described by a binary mask for 1433 pertinent dictionary words and an integer in {0..6} representing its type.
+The Cora dataset consists of 2708 scientific publications represented as nodes interconnected by 5429
+reference links represented as edges. Each paper is described by a binary mask for 1433 pertinent
+dictionary words and an integer in {0..6} representing its type.
 First we download the Cora dataset and convert it to a valid binary representation via our built-in Cora downloader.
 
 .. code-block:: python
```

Comment:
We don't need this one:
```python
>>> Cora("/tmp/cora/")
```
looks like you are using in memory clients below instead of a distributed setup.

Reply:
This is to generatte the Cora dataset so we can use it in the example.

Reply:
We generate it anyway later, right?

Reply:
We use distro client now, in next PR.